commit ba58ab6dba12ac80e221edc98ecd5b998030ca4a (refs/changes/87/3087/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-12-11T12:38:14-08:00 (1 year, 10 months ago)
    
    TOOLS-1949 sdcnode updates for Dec 2017 node.js security releases
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
    Approved by: Josh Wilsdon <josh@wilsdon.ca>

diff --git a/Makefile b/Makefile
index 3e67c0b..e127d9b 100644
--- a/Makefile
+++ b/Makefile
@@ -24,7 +24,7 @@ JSSTYLE_FLAGS	 = -f tools/jsstyle.conf
 SMF_MANIFESTS_IN = smf/manifests/docker.xml.in
 CLEAN_FILES += ./node_modules
 
-NODE_PREBUILT_VERSION=v4.8.5
+NODE_PREBUILT_VERSION=v4.8.7
 ifeq ($(shell uname -s),SunOS)
 	NODE_PREBUILT_TAG=zone
 	# Allow building on other than image sdc-minimal-multiarch-lts@15.4.1.
